### DISCUSSION TITLE
AI女團Eternity走進韓國K-pop流行樂壇

### DISCUSSION THREAD

**Discussion by liangyinglly** [1101-GS3073-C/Python-Community-GS3073-C#694] [General]
不知道大家有沒有看過韓國AI女團Eternity的ＭV，我覺得還是能看出他們是虛擬合成的，以下附上影片連結
https://www.youtube.com/watch?v=XQMg8-Ku3JI

這個由AI產生的虛擬人像是由朴智恩（Park Jieun）所創造的，他宣稱AI女團不會受到輿論壓力，不會有維持身材的壓力等，我覺得這些出發點都是很好的，如果技術再不斷地進步，應該就能跟真人看不出任何差別了吧，你們覺得創造這樣的虛擬女團是拯救了韓國的練習生免於社會輿論壓力呢還是會搶了他們的工作，是件該擔心的事呢？

文章連結：https://www.thenewslens.com/article/178107/fullpage

**Comment by Vincent20030314** [comment]
雖然保障他們，但是可能導致該產業門檻降低。

**Comment by HsuChen88** [comment]
可是我們想看的不是機器人啊!!

**Comment by yjpxi** [comment]
如果虛擬偶像變得越來越逼真且受歡迎，它們可能會取代部分真人藝人的工作機會，特別是在娛樂產業中，但是大家對虛擬偶像的連結可能沒有那麼強。
